tests/helper/helper_telemetry.go

```diff
@@ -0,0 +1,38 @@
+package helper
+
+import (
+	"encoding/json"
+	_ "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-developer/odo/pkg/preference"
+	"github.com/redhat-developer/odo/pkg/segment"
+	"io/ioutil"
+	"os"
+)
+
+//CreateTelemetryDebugFile creates a temp file to use for debugging telemetry.
+//it also sets up envs and cfg for the same
+func CreateTelemetryDebugFile() {
+	Expect(os.Setenv("ODO_DISABLE_TELEMETRY", "false")).NotTo(HaveOccurred())
```

Comment:
Please use `segment.DisableTelemetryEnv` instead of the env name. Same for other env names in this file.

Reply:
ok

Reply:
Whey are you setting this env var in this function? The function name does not prepare to this